content/applications/finance/fiscal_localizations/chile.rst

```diff
@@ -2,898 +2,998 @@
+  | **Hint:** Make sure the field :guilabel:`Document Type` and :guilabel:`VAT` are set in the
+    customer and in the main company.
+- | **Error:** `Usuario sin permiso de envio.`
+  | **Hint:** This error indicates that most likely, your company has not passed the `Certification
+    process <https://www.sii.cl/factura_electronica/factura_mercado/proceso_certificacion.htm>`_ in
+    the :abbr:`SII (Servicio de Impuestos Internos)` - Sistema de Facturación de Mercado. If this is
+    the case, please contact your Account Manager or Customer Support as this certification is not
+    part of the Odoo services, but we can give you some alternatives. If you already passed the
+    certification process, this error appears when a user different from the owner of the
+    certificate is trying to send :abbr:`DTE (Documentos Tributarios Electrónicos)` files to the
+    :abbr:`SII (Servicio de Impuestos Internos)`.
+- | **Error:** `CARATULA`
+  | **Hint:** There are just five reasons why this error could show up and all of them are related
+    to the *Caratula* section of the XML:
+
+    - The company's RUT number is incorrect or missing.
+    - The certificate owner RUT number is incorrect or missing.
+    - The SII's RUT number (this should be correct by default).
+    - The resolution date.
+    - The resolution number.
 
 .. _chile/credit-notes:
 
-Credit Notes
+Credit notes
 ------------
 
 When a cancellation or correction is needed over a validated invoice, a credit note must be
-generated. It is important to consider that a CAF file is required for the Credit Note,
-which is identified as document 64 in the SII.
+generated. It is important to consider that a :abbr:`CAF (Folio Authorization Code)` file is
+required for the credit note, which is identified as :guilabel:`Document Type` :guilabel:`61` in the
+:abbr:`SII (Servicio de Impuestos Internos)`. Please refer to the :ref:`CAF section
+<chile/caf-documentation>` for more information on the process to load the :abbr:`CAF (Folio
+Authorization Code)` on each document type.
 
-.. image:: chile/Chile40.png
-   :align: center
+.. image:: chile/credit-note-document-type.png
    :alt: Creation of CAF for Credit notes.
+   :align: center
 
-
-.. tip::
-   Refer to the :ref:`CAF section <chile/caf-documentation>` where we described the process to load
-   the CAF on each document type.
-
-Use Cases
+Use cases
 ~~~~~~~~~
 
-Cancel Referenced document
+Cancel referenced document
 **************************
 
-In case you need to cancel or invalid an Invoice, use the button Add Credit note and select Full
-Refund, in this case the SII reference Code is automatically set to: Anula Documento de referencia.
+In case you need to cancel or invalid an invoice, use the button :guilabel:`Add Credit note` and
+select :guilabel:`Full Refund`, in this case the :abbr:`SII (Servicio de Impuestos Internos)`
+reference code is automatically set to :guilabel:`Anula Documento de referencia`.
```

Comment:
Is this where I'd find an invoice? I don't see the "Full Refund" option but I might be missing some configuration

```suggestion
In case you need to cancel or invalid an invoice, navigate to :menuselection:`Accounting -->
Customers --> Invoices` and select the desired invoice. Then, use the button :guilabel:`Add Credit
Note` and select :guilabel:`Full Refund`, in this case the :abbr:`SII (Servicio de Impuestos
Internos)` reference code is automatically set to :guilabel:`Anula Documento de referencia`.
```
![image](https://github.com/odoo/documentation/assets/23611711/3cd1a985-9bc9-41e3-b225-928828dca962)